roles/install_openstack_ca/tasks/main.yml

```diff
@@ -14,13 +14,17 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
-- name: Get CA bundle data
+- name: Get CA bundle data with retries
   environment:
     KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
     PATH: "{{ cifmw_path }}"
   ansible.builtin.command:
-    cmd: 'oc get secret combined-ca-bundle -n openstack -o "jsonpath={.data.tls-ca-bundle\.pem}" --ignore-not-found'
+    cmd: 'oc get secret combined-ca-bundle -n openstack -o "jsonpath={.data.tls-ca-bundle\.pem}"'
+  retries: 10
+  delay: 3
+  until: ca_bundle_data.rc == 0
   register: ca_bundle_data
+  ignore_errors: true
 
 - name: Get CA bundle
   when: ca_bundle_data.stdout | length > 0
```

Comment:
Are we sure that if it fails the stdout is empty? (I saw oc sometimes using stdout for errors too, instead of stderr). Isn't safer to check here if `rc == 0`?

Reply:
you're right, that condition worked for the previous version, since it had `--ignore-not-found`, updating it now